src/main/java/techcourse/myblog/domain/Article.java

```diff
@@ -1,4 +1,56 @@
 package techcourse.myblog.domain;
 
+import org.apache.commons.lang3.StringUtils;
+
 public class Article {
+    private int articleId;
```

Comment:
`Article`의 식별자니깐 아래의 변수 이름은 어떨까요?

```suggestion
    private int id;
```